vividus-extension-selenium/src/main/java/org/vividus/selenium/RemoteWebDriverFactory.java

```diff
@@ -30,11 +30,11 @@
     @Override
     public RemoteWebDriver getRemoteWebDriver(URL url, Capabilities capabilities)
     {
-        if (WebDriverManager.isIOS(capabilities))
+        if (GenericWebDriverManager.isIOS(capabilities))
```

Comment:
Why it can't be part of mobile-app plugin?

If you starting web profile you will have RemoteDriverFactory if you starting the mobile app profile you will have an available factory for ios/android

Reply:
Discussed privately